app/scrapers/__init__.py

```diff
@@ -28,6 +29,7 @@
     't': Youtube(),
     'p': Parsijoo(),
     'm': Mojeek(),
+    'h': QihooSo(),
```

Comment:
#327 must be fixed before we propagate more workarounds.

Reply:
No need of this.

Reply:
Please change "h" to "qihooso"